README.md

```diff
@@ -4,15 +4,62 @@ We are committed to providing an open platform for developers to build upon.
 
 While the Cards themselves are stored on the Ethereum blockchain (or in the Starkware rollup) we support an API that provides more detailed information.
```

Comment:
This is a big first step toward clarification. I think this should be at some point broken down into three distinct sections for authentication:
1. API Key (including unauthenticated) 
2. JWT
3. OAuth